dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -661,6 +662,35 @@ def test_sample_size_passed_to_profile(self, *mocks):
         profiler._sampling_ratio = 0.2
         self.assertEqual(10000, update_mock.call_args[0][1])
 
+    def test_index_overlap_resolved_for_update_profile(self):
```

Comment:
probably don't need to compute any of the compilers, can mock them out so we don't do data labeling, etc

Reply:
Deja Vu, my bad